### DISCUSSION TITLE
J shortcut

### DISCUSSION THREAD

**Discussion by pax0707** [FreshRSS/FreshRSS#5948] [Q&A]
When I use J after reading the last article in the category sometimes it will go to the next unread category. Love this but can't reproduce it reliably. Most of the time it will stop on the last article in the category.

Bug or a feature? What is the expected behavior?

I know I can use Alt+J but just spamming J is easier and faster when going through freshly refreshed timeline with a few articles per category.

**Comment by Alkarex** [comment]
Probably a bug that it does not work in all cases. Debugging + PR welcome

**Comment by pax0707** [comment]
Got it @Alkarex:

1) If you 1st click on a category J will work till you reach the end of displayed posts in the category. After that, it will not progress to the next category. 

2) If you 1st click on a feed within a category J will progress to the next category, after no more articles in the current one.




I mostly read articles by category so I always run into scenario 1.
